src/modules/word/index.ts

```diff
@@ -315,4 +316,83 @@ export class WordModule {
       })
     );
   }
+
+  /**
+   * Returns a random sample of random or optionally specified length.
+   *
+   * @param options The expected length of the word or the options to use.
+   * @param options.length The expected length of the word.
+   * @param options.strategy The strategy to apply when no words with a matching length are found.
+   *
+   * Available error handling strategies:
+   *
+   * - `fail`: Throws an error if no words with the given length are found.
+   * - `shortest`: Returns any of the shortest words.
+   * - `closest`: Returns any of the words closest to the given length.
+   * - `longest`: Returns any of the longest words.
+   * - `any-length`: Returns a word with any length.
+   *
+   * Defaults to `'any-length'`.
+   *
+   * @example
+   * faker.word.sample() // 'incidentally'
+   * faker.word.sample(5) // 'fruit'
+   *
+   * @since 8.0.0
+   */
+  sample(
+    options:
+      | number
+      | {
+          length?: number | { min: number; max: number };
+          strategy?: 'fail' | 'closest' | 'shortest' | 'longest' | 'any-length';
+        } = {}
+  ): string {
+    const wordMethods = [
+      this.adjective,
+      this.adverb,
+      this.conjunction,
+      this.interjection,
+      this.noun,
+      this.preposition,
+      this.verb,
+    ];
+    this.faker.helpers.shuffle(wordMethods);
+
+    for (const randomWordMethod of wordMethods) {
+      try {
+        return randomWordMethod(options);
+      } catch {
+        // catch missing locale data potentially required by randomWordMethod
+        continue;
+      }
+    }
+
+    throw new FakerError(
+      'No matching word data available for the current locale'
+    );
+  }
+
+  /**
+   * Returns a string containing a number of space separated random words.
+   *
+   * @param count Number of words. Defaults to a random value between `1` and `3`.
+   *
+   * @example
+   * faker.word.words() // 'almost'
+   * faker.word.words(5) // 'before hourly patiently dribble equal'
+   *
+   * @since 8.0.0
+   */
+  words(count?: number): string {
+    if (count == null) {
+      count = this.faker.datatype.number({ min: 1, max: 3 });
+    }
+
+    const words: string[] = [];
+    for (let i = 0; i < count; i++) {
+      words.push(this.sample());
+    }
+    return words.join(' ');
```

Comment:
Can we rewrite this to this?

```suggestion
    return Array.from({ lenght: count }, () => this.sample()).join(' ');
```